src/plugins/data/public/ui/query_editor/query_editor.tsx

```diff
@@ -92,6 +93,9 @@ export default class QueryEditorUI extends Component<Props, State> {
   private services = this.props.opensearchDashboards.services;
   private componentIsUnmounting = false;
   private queryEditorDivRefInstance: RefObject<HTMLDivElement> = createRef();
```

Comment:
tbh I think I carried this over from the query string input component and it was used for attaching suggestions. I wonder then if we should just delete this then since i think the header ref more or less solves it.

Reply:
removed the handleListUpdate related stuff in 5b3e3589cc3